Tests/tbDEXTests/Protocol/Models/Messages/OrderStatusTests.swift

```diff
@@ -9,62 +9,26 @@ final class OrderStatusTests: XCTestCase {
     let pfi = try! DIDJWK.create(keyManager: InMemoryKeyManager())
 
     func test_init() {
-        let orderStatus = createOrderStatus(from: did.uri, to: pfi.uri)
+        let orderStatus = DevTools.createOrderStatus(from: pfi.uri, to: did.uri)
```

Comment:
i know this is a small detail change but i appreciate it because it's more accurate :D